8_child_illness.do

```diff
@@ -35,7 +35,7 @@ order *,sequential  //make sure variables are in order.
 	    foreach var of varlist h12a-h12x {
 	    local lab: variable label `var' 	   
         replace `var' = . if ///
-	    regexm("`lab'","(oth(er)|shop|pharmacy|market|kiosk|relative|friend|church|drug|addo|rescuer|trad|unqualified|stand|cabinet|ayush|^na|oth.|homeopath)") ///
+	    regexm("`lab'","(priv(ate))|(oth(er)|shop|pharmacy|market|kiosk|relative|friend|church|drug|addo|rescuer|trad|unqualified|stand|cabinet|ayush|^na|oth.|homeopath)") ///
```

Comment:
Actually, I would suggest to first compare the provider for this survey with the list given in "Diarrhea_fever_ari1_ari2_treatment_list.xlsx" for decision, then add the exact name of the provider as key words if you need to do changes. This excel file, with all other supporting materials can be found in Onedrive.
"(priv(ate))" and doesn't appear in Pakistan2006 h12a-h12x nor h32a-h32x, thus I suppose it won't do changes. Also, adding this key words might affect other survey result unintentionally, thus we usually avoid adding ambigous key words like "private", unless we are very confident with the juedgement, such as "shop".